update: iOS accessibility tree sync improvement (#324)
JetBrains/kotlin-multiplatform-dev-docs: topics/whats-new/whats-new-compose-180.md

Compose Multiplatform %composeEapVersion% introduces accessibility support for right-to-left languages,
including proper text direction handling for gestures.

#### Loading accessibility tree on demand

Instead of setting a specific mode of syncing the Compose semantic tree with the iOS accessibility tree,

> you can now rely on Compose Multiplatform to do this lazily:

Reviewer: ```suggestion
you can now rely on Compose Multiplatform to handle this process lazily:
```

> the tree is loaded completely after the first request from the iOS accessibility engine

Reviewer: ```suggestion
the tree is fully loaded after the first request from the iOS accessibility engine
```

and is disposed of when the screen reader stops interacting with it.

This allows fully supporting iOS Voice Control, VoiceOver,
and other accessibility tools that rely on the accessibility tree.

> The `AccessibilitySyncOptions` class which was [used to configure accessibility tree sync](compose-ios-accessibility.md#choose-the-tree-synchronization-option)
> is removed as it is no longer necessary.

Reviewer: ```suggestion
The `AccessibilitySyncOptions` class, which was [used to configure accessibility tree sync](compose-ios-accessibility.md#choose-the-tree-synchronization-option),
has been removed as it is no longer necessary.
```
